2주차/FE/문민제/animation-webpage_2/css/style.css

```
@@ -0,0 +1,132 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+a{
+    text-decoration: none;
+}
+
+p{  
+    font-size: 20px;
+    font-style: bold;
+    color: snow;
+}
+
+body{
+    background: none;
+}
+
+h2{
+    color: wheat;
+}
+
+.hidden{
+    display: none;
+}
+
+.container{
+    width: 100%;
+    height: 100vh;
+    background: linear-gradient(rgba(0,0,0,0.5),rgba(0,0,0,0.2)),url(../img/kimetsu2.jpg);
+}
+
+.container .title{
+    width: fit-content;
+    height: fit-content;
+    margin: 0 auto;
+    display: flex;
+    justify-content: left;
+    position: absolute;
+    left: 300px;
+    top: 27%;
+    transform: none;
+    text-align: center;
+}
+
+.container .sub{
+    width: fit-content;
+    height: fit-content;
+    margin: 0 auto;
+    display: flex;
+    justify-content: left;
+    position: absolute;
+    left: 200px;
+    top: 25%;
+    transform: none;
+    text-align: center;    
+}
+
+.container .title h1{
+    color: wheat;
+    font-size: 85px;
+}
+
+.container .header {
+    width: 80%;
+    height: 80px;
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    position: static;
+}
+
+.container .detail{
+    margin-left: 800px;
+    display: inline-block;
+    position: static;
+}
+
+.container .detail button{
+    background: none;
+    border: none;
+    font-size: 18px;
+    color: white;
+    cursor: pointer;
+    padding: 8px;
+}
+
+.container .detail .genre{
+    display: inline-block;
+    position: absolute;
+    background-color: rgba(0,0,0,0.5);
+}
+
+.container .detail .genre, p{
+    color:snow;
+    font-size: 18px;
+}
+
+.img-list{
+    position: absolute;
+    bottom: 7%;
+}
+
+.img-list .img-box{
+    width: 300px;
+    height: 430px;
+    background-color: none;
+    float: left;
+    margin-left: 110px;
+    margin-right: 60px;
+    overflow: hidden;
+    cursor: pointer;
+}
+
+.image1{
+    height: 100%;
+}
+
+.image2{
+    height: 100%;
+}
+
+.image3{
+    height: 100%;
+}
+
+.image4{
+    height: 100%;
+}
```

아래와 같은 방법으로 바꿔봐도 괜찮을 것 같습니다!
```suggestion
.image1, .image2, .image3, .image4 {
    height: 100%;
}
```

또는 image1, image2, image3, image4의 차이가 없다면 같은 class name으로 사용해도 괜찮을 것 같습니다 :)